```
[#x1B[34m2020-11-18 12:15:46,697#x1B[0m] {#x1B[34mapplication.py:#x1B[0m297} INFO#x1B[0m - Reporting errors#x1B[0m
/home/circleci/project/astronomer/migration_spinner/command_line.py:34:36: F523 '...'.format(...) has unused arguments at position(s): 1
[#x1B[34m2020-11-18 12:15:46,698#x1B[0m] {#x1B[34mapplication.py:#x1B[0m303} INFO#x1B[0m - Found a total of 1 violations and reported 1#x1B[0m
```

--- a/astronomer/migration_spinner/command_line.py
+++ b/astronomer/migration_spinner/command_line.py
@@ -31,8 +31,9 @@ def spinner(timeout):
                 logging.info('Current heads: {}'.format(db_heads))
                 break
             elif ticker >= timeout:
-                raise TimeoutError("There are still unapplied migrations after {} "
-                                   "seconds".format(ticker, timeout))
+                raise TimeoutError(
+                    "There are still unapplied migrations after {} seconds".format(timeout)
+                )
             ticker += 1
             time.sleep(1)
             logging.info('Waiting for migrations... {} second(s)'.format(ticker))

